lib/watir/locators/element/locator.rb

```diff
@@ -155,6 +157,8 @@ def fetch_value(element, how)
           case how
           when :text
             element.text
+          when :visible_text
```

Comment:
This can be done with `when :text, :visible_text`

Reply:
Won't :text and :visible_text be different once we merge with #686?

Reply:
lol. Yes. Disregard.